package.json

```diff
@@ -23,29 +23,33 @@
   "bugs": {
     "url": "https://github.com/bryanpizzillo/r4rimporter/issues"
   },
+  "engines": {
+    "node": "^16"
+  },
   "homepage": "https://github.com/bryanpizzillo/r4rimporter#readme",
   "dependencies": {
-    "@octokit/rest": "^15.2.6",
-    "axios": "^0.18.0",
-    "config": "^1.30.0",
-    "elastic-tools": "git+https://github.com/nciocpl/elastic-tools.git#1.0.0",
-    "elasticsearch": "^14.2.2",
-    "gray-matter": "^3.1.1",
+    "@elastic/elasticsearch": "^7.17.0",
+    "@octokit/rest": "^19.0.4",
+    "axios": "^0.27.2",
+    "config": "^3.3.8",
+    "elastic-tools": "git+https://github.com/nciocpl/elastic-tools.git#v2.0.0",
```

Comment:
Not for this PR, but we should really publish these as packages as we can do that now.

Reply:
Added as todo items on the web apis project
- [loader-pipeline](https://github.com/NCIOCPL/loader-pipeline/issues/4)
- [elastic-tools](https://github.com/NCIOCPL/elastic-tools/issues/10)